build/Dockerfile

```diff
@@ -1,8 +1,20 @@
+FROM openshift/origin-release:golang-1.10
+
+RUN mkdir -p /go/src/github.com/embercsi/ember-csi-operator/
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+COPY . /go/src/github.com/embercsi/ember-csi-operator/
```

Comment:
nit: To speed things up we should have a `.dockerignore` file that skips the `.git` and  `build/_output/bin` directories.

Reply:
good idea, added